vulnerabilities/management/commands/import.py

```diff
@@ -21,16 +21,14 @@
 #  VulnerableCode is a free software code scanning tool from nexB Inc. and others.
 #  Visit https://github.com/nexB/vulnerablecode/ for support and download.
 
-from datetime import datetime
 import traceback
 
 from django.core.management.base import BaseCommand
 from django.core.management.base import CommandError
 
-from vulnerabilities.models import Importer
-from vulnerabilities.importer_yielder import IMPORTER_REGISTRY
+from vulnerabilities.importers import IMPORTER_REGISTRY
+from vulnerabilities.importers import importer_mapping
```

Comment:
There should be only one IMPORTER_REGISTRY

Reply:
Fixed